chromium_src/ios/components/webui/web_ui_url_constants.h

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_IOS_COMPONENTS_WEBUI_WEB_UI_URL_CONSTANTS_H_
+#define BRAVE_CHROMIUM_SRC_IOS_COMPONENTS_WEBUI_WEB_UI_URL_CONSTANTS_H_
+
+extern const char kChromeUIUntrustedScheme[];
```

Comment:
Do these new scheme URLs need to be in a `chromium_src` override? It seems like we should have a `brave_web_ui_url_constants.h` file which is included here (and where we need it) so we don't need to pull in a `chromium_src` review every time this changes?